ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -56,19 +57,7 @@ const styles = reactStyles({
   }
 });
 
-const freeCreditLimitOptions = [
-  {label: '$300', value: 300},
-  {label: '$350', value: 350},
-  {label: '$400', value: 400},
-  {label: '$450', value: 450},
-  {label: '$500', value: 500},
-  {label: '$550', value: 550},
-  {label: '$600', value: 600},
-  {label: '$650', value: 650},
-  {label: '$700', value: 700},
-  {label: '$750', value: 750},
-  {label: '$800', value: 800}
-];
+const freeCreditLimitDefaultOptions = new Set([300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800]);
```

Comment:
can't hardcode the options list because we also need to include the current override value, if one exists and is not one of these

Reply:
nit: consider [fp.range](https://lodash.com/docs/4.17.15#range).

Reply:
I considered this, but was surprisingly unsuccessful: https://github.com/all-of-us/workbench/commits/joel/RW-4956-range

Reply:
Fixed, after Greg's surprising suggestion to randomly rearrange the params until it works.  Apparently this is a "fun quirk" of fp-lodash.  Added comment to explain.